Src/WitsmlExplorer.Frontend/components/ContentViews/TrajectoryView.tsx

```diff
@@ -80,7 +81,13 @@ export const TrajectoryView = (): React.ReactElement => {
     { property: "md", label: "md", type: ContentType.Number },
     { property: "incl", label: "incl", type: ContentType.Number },
     { property: "azi", label: "azi", type: ContentType.Number },
-    { property: "tvd", label: "tvd", type: ContentType.Number }
+    { property: "tvd", label: "tvd", type: ContentType.Number },
+    { property: "dls", label: "dls", type: ContentType.Number },
+    { property: "mtf", label: "mtf", type: ContentType.Number },
+    { property: "gtf", label: "gtf", type: ContentType.Number },
+    { property: "dispNs", label: "dispNs", type: ContentType.Number },
+    { property: "dispEw", label: "dispEw", type: ContentType.Number },
+    { property: "vertSect", label: "vertSect", type: ContentType.Number }
```

Comment:
As mentioned in the same [comment](https://github.com/equinor/witsml-explorer/issues/183#issuecomment-1686114631), they want the other properties to be added after. So seems like they want all the properties to be added to the listView.

Reply:
I clarified it with Asbjørn and these additional properties should only be in the modify modal, not in listView.  I'll add a comment the next time.